tests/bluetooth/tester/src/audio/btp/btp_csis.h

```diff
@@ -33,7 +33,7 @@ struct btp_csis_get_member_rsi_rp {
 	uint8_t rsi[BT_CSIP_RSI_SIZE];
 } __packed;
 
-#define BTP_CSIS_ENC_SIRK_TYPE			0x04
+#define BTP_CSIS_SIRK_TYPE			0x04
 struct btp_csis_sirk_type_cmd {
```

Comment:
Consider naming both to use the name used by BTP which is "Set SIRK type"


```suggestion
#define BTP_CSIS_SET_SIRK_TYPE			0x04
struct btp_csis_set_sirk_type_cmd {
```